lib/tasks/email.rake

```diff
@@ -26,5 +27,15 @@ namespace :email do
         Airbrake.close
       end
     end
+
+    namespace :second do
+      desc "Collect all registration that expires in X weeks and sends a second email reminder"
```

Comment:
I think it will be clearer to say

```suggestion
      desc "Send second email reminder to all registrations expiring in X days (default is 14)"
```

I know this does not match the description for the first one. But it also talks in weeks when we set the value in days. So I'd also suggest / be happy with the description for the first reminder task to be updated as well in this PR to

```ruby
      desc "Send first email reminder to all registrations expiring in X days (default is 28)"
```